Formula/himalaya.rb

```diff
@@ -4,6 +4,7 @@ class Himalaya < Formula
   url "https://github.com/soywod/himalaya/archive/v0.7.1.tar.gz"
   sha256 "79ad75e765a5298c5c597beb175cec1d2982eead2172bc78daa7a9f159c99861"
   license "MIT"
+  revision 1
```

Comment:
A revision bump isn't necessary for completions, as it implies that users will have to upgrade their installation with `brew upgrade`. That would be too much for a small improvement.

Reply:
I understand, the idea was to not wait for the next release of Himalaya for having completions generated by brew at installation.

Reply:
I get what you mean. In that case, anyone wishing to use the completions may do a `brew reinstall`.